pyvista/plotting/plotting.py

```diff
@@ -90,22 +95,63 @@ def __init__(self, shape=(1, 1), border=None, border_color='k',
                 border = False
 
         # add render windows
-        self.renderers = []
         self._active_renderer_index = 0
-        assert_str = '"shape" should be a list or tuple'
-        assert isinstance(shape, collections.Iterable), assert_str
-        assert shape[0] > 0, '"shape" must be positive'
-        assert shape[1] > 0, '"shape" must be positive'
-        self.shape = shape
-        for i in reversed(range(shape[0])):
-            for j in range(shape[1]):
-                renderer = pyvista.Renderer(self, border, border_color, border_width)
-                x0 = i/shape[0]
-                y0 = j/shape[1]
-                x1 = (i+1)/shape[0]
-                y1 = (j+1)/shape[1]
-                renderer.SetViewport(y0, x0, y1, x1)
-                self.renderers.append(renderer)
+        self.renderers = []
+
+        if isinstance(shape, str):
+
+            if '|' in shape:
+                n = int(shape.split('|')[0])
+                m = int(shape.split('|')[1])
+                rangen = reversed(range(n))
+                rangem = reversed(range(m))
+            else:
+                m = int(shape.split('/')[0])
+                n = int(shape.split('/')[1])
+                rangen = range(n)
+                rangem = range(m)
+
+            if n>=m:
+                xsplit = m/(n+m)
+            else:
+                xsplit = 1-n/(n+m)
+
+            if rcParams['multi_rendering_splitting_position']:
```

Comment:
Can you add `splitting_position` as an argument here with a default to None and if None, grab the value from the `rcParams`?